pkg/generator/data/main.java.tpl

```diff
@@ -0,0 +1,54 @@
+import java.io.BufferedReader;
```

Comment:
Please add the license header as well.

Reply:
The message "Copyright 2023 API Testing Authors." should be added to the .tpl file, do I understand correctly?

Reply:
Yes, each source code file should have the license header.